subgraph.yaml

```diff
@@ -15,6 +15,7 @@ dataSources:
       entities:
         - Delegation
         - Coin
-      transactionHandlers:
-        - handler: handleTx
+      messageHandlers:
+        - type_url: /cosmos.staking.v1beta1.MsgDelegate
```

Comment:
We should probably drop the slash at the beginning of each message type. @lavabyrd We would need to update the Firehose filter too.